<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>poid</class>
 <widget class="QWidget" name="poid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>545</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>poid</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 189, 182);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>441</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Information personnelles</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_suite">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Suite</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>271</width>
       <height>186</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Prénom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_prenom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Sexe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_femme">
          <property name="text">
           <string>Femme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_homme">
          <property name="text">
           <string>Homme</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_age">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Poids (kg)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_poid">
          <property name="minimum">
           <double>15.000000000000000</double>
          </property>
          <property name="maximum">
           <double>300.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>66.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Taille (m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_taille">
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>3.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.700000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Calcul du poids Idéal</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>260</y>
       <width>391</width>
       <height>86</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_deivne">
        <property name="text">
         <string>Poids idéal formulde de devine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_lorentz">
        <property name="text">
         <string>Poids idéal formule de lorentz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sansage">
        <property name="text">
         <string>Poids idéal formule de lorentz en fonction de l'age</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
